paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -55,10 +56,11 @@ def add_subparser(subparsers):
             required=True,
         ).completer = lazy_choices_completer(list_instances)
         status_parser.add_argument(
-            '-c', '--cluster',
-            help='The PaaSTA cluster that has the service you want to %s. Like norcal-prod' % lower,
-            required=True,
-        ).completer = lazy_choices_completer(utils.list_clusters)
+            '-c', '--clusters',
```

Comment:
Using nargs would be nice but this would introduce a third difference between this command and others - the existing --clusters commands use comma-separated lists. I'm happy to change those also but it seems like a bit of a big shift in usage.